library/Freckle/Test/Yesod/JsonAssertions.hs

```diff
@@ -0,0 +1,93 @@
+module Freckle.Test.Yesod.JsonAssertions
+  ( requireJSONResponse
+  , bodyMatchesAsJson
+  , bodyArrayShouldSatisfyObjectProperties
+  , bodyDeepPermutationAsJson
+  , bodyEditEqualsAsJson
+  , bodyEqualsAsJson
```

Comment:
These are are legacy functions and we prefer to use the functions from `hspec-expectations-json` directly instead. Let's not bring that baggage over please.

Reply:
https://github.com/freckle/megarepo/pull/29168 removes all of these, except `bodyArrayShouldSatisfyObjectProperties` which will require a tad more thought.